### PR TITLE
Update openstack overview page

### LINKED ISSUE #3201: Updates to Openstack overview page
https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit?disco=AAAABxixEYo&ts=5b0045b1

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Update page as per [copy doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit?ts=5b0045b1#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/openstack>
- Make sure page matches the [copy doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit?ts=5b0045b1#) - please check off comments as you verify these changes

## Issue / Card

Fixes #3201

### CODE REVIEW

**Comment by frankiegranato** [comment]
We need to add a link to David Calle's new landing page https://github.com/canonical-websites/www.ubuntu.com/pull/3203 to the "Learn more about OpenStack" section. I'm going to talk to @pmahnke about this.

**Comment by pmahnke** [comment]
@steverydz and @frankiegranato I think that is the right link - /engage/vmware-to-openstack, I added it to the copy doc

File: templates/openstack/index.html [hunk 1]
```diff
@@ -1,24 +1,83 @@
...
+            <li>High availability</li>
+            <li>Upgrades to future versions of OpenStack on demand</li>
+          </ul>
+          <p>$80k with two week delivery on certified hardware.</p>
+          <p><a class="p-button--positive" href="/download/cloud">Get OpenStack&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+      <div class="col-4 p-card">
+        <h4 class="p-card__title">Cloud Build Plus</h4>
+        <div class="p-card__body">
```
Comment (pmahnke) [inline templates/openstack/index.html]: should be `p-card__content`

File: templates/openstack/index.html [hunk 2]
```diff
@@ -1,24 +1,83 @@
...
+            <li>Upgrades on demand</li>
+            <li>High availability</li>
+          </ul>
+          <p>$120k with two week delivery on certified hardware.</p>
+          <p><a class="p-button--positive" href="/download/cloud">Migrate from VMWare&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+      <div class="col-4 p-card">
+        <h4 class="p-card__title">Fully Managed - Bootstack</h4>
+        <p>Full remote management of your Canonical OpenStack at half the price of VMware.</p>
```
Comment (pmahnke) [inline templates/openstack/index.html]: should be `p-card__content` in a div

File: templates/openstack/index.html [hunk 3]
```diff
@@ -1,24 +1,83 @@
...
+          <ul>
+            <li>Hardware recommendations and sizing</li>
+            <li>Fixed-price deployment</li>
+            <li>Reference architecture with standard storage and networking options</li>
+            <li>Hyperconverged and containerised control plane</li>
+            <li>High availability</li>
+            <li>Upgrades to future versions of OpenStack on demand</li>
+          </ul>
+          <p>$80k with two week delivery on certified hardware.</p>
+          <p><a class="p-button--positive" href="/download/cloud">Get OpenStack&nbsp;&rsaquo;</a></p>
```
Comment (pmahnke) [inline templates/openstack/index.html]: the link should be /openstack/consulting (you need to look at the beta to guess the urls)

File: templates/openstack/index.html [hunk 4]
```diff
@@ -1,24 +1,83 @@
...
+            <li>Hardware recommendations and sizing</li>
+            <li>Fixed price deployment</li>
+            <li>Custom architecture with containerised or standalone control plane</li>
+            <li>SAN and SDN integration</li>
+            <li>Workload analysis and migration plan</li>
+            <li>Upgrades on demand</li>
+            <li>High availability</li>
+          </ul>
+          <p>$120k with two week delivery on certified hardware.</p>
+          <p><a class="p-button--positive" href="/download/cloud">Migrate from VMWare&nbsp;&rsaquo;</a></p>
```
Comment (pmahnke) [inline templates/openstack/index.html]: the link should be /engage/vmware-to-openstack

File: templates/openstack/index.html [hunk 5]
```diff
@@ -1,24 +1,83 @@
...
+        <p>Full remote management of your Canonical OpenStack at half the price of VMware.</p>
+        <ul>
+          <li>24/7 monitoring and management</li>
+          <li>EU and US regulatory compliance options</li>
+          <li>Telco NFV options</li>
+          <li>Finance sector options</li>
+          <li>SDN options</li>
+        </ul>
+        <p>$500 per server per month or $5,000 per server per year, all in. The best way to start with OpenStack. Recognised by analysts as the best value private cloud.</p>
+        <p><a class="p-button--positive" href="/cloud/managed-cloud">Fully Managed OpenStack&nbsp;&rsaquo;</a></p>
```
Comment (pmahnke) [inline templates/openstack/index.html]: the link should be /openstack/managed

**Comment by steverydz** [comment]
@pmahnke @frankiegranato Updates made

File: templates/openstack/index.html [hunk 6]
```diff
@@ -109,32 +174,17 @@ <h3 class="p-matrix__title p-heading--five">Model and deploy with Juju</h3>
...
-  <div class="row">
-    <div class="col-12">
-      <div class="u-embedded-media">
-        <iframe class="u-embedded-media__element" src="https://player.vimeo.com/video/202448226?color=ffffff&amp;title=0&amp;byline=0&amp;portrait=0" frameborder="0" allowfullscreen></iframe>
-      </div>
-    </div>
-  </div>
-</section>
-
 <section class="p-strip--light is-deep is-bordered">
```
Comment (pmahnke) [inline templates/openstack/index.html]: @steverydz either this needs to be just a p-strip, or voidbridge's logo needs to up updated to be transparent.

Comment (steverydz) [reply templates/openstack/index.html]: @pmahnke I've made it just a p-strip - I had a look for a transparent logo but couldn't find one - even on their website

**Comment by anasereijo** [comment]
- can we have some padding between the body copy and the 3 cards? I tried 24px and looks better (see below)
screen shot 2018-05-29 at 11 06 37

- "The number one cloud OS" should be an h2

- again, can we have padding between body copy and matrix? see below
screen shot 2018-05-29 at 11 11 14

- "Fully managed OpenStack solution" section LOGOS: we have some empty <li> can we please remove them? Also, can we fix the logo sizes? (see below) Let me know if you need me to design this,

screen shot 2018-05-29 at 11 13 39

thanks

**Comment by steverydz** [comment]
Closing as superseded by #3281

**Pull request closed without merging**